src/constants.h

```diff
@@ -68,16 +68,16 @@
  *                             -----------
  *                            |           |
  *                            |           |
- *                          0 |           | 2
+ *                          0 |           | 1
  *                            |           |
  *                            |           |
  *                             -----------
- *                            0     1     1
+ *                            0     2     1
  */
 #define NUM_SURFACES 4
 #define SURFACE_X_MIN 0
-#define SURFACE_Y_MIN 1
-#define SURFACE_X_MAX 2
+#define SURFACE_X_MAX 1
+#define SURFACE_Y_MIN 2
 #define SURFACE_Y_MAX 3
 
 #define NUM_CORNERS 4
```

Comment:
I figured that for 3D MOC, this will be modified to include the z-surfaces and corners/edges that include z.